docs/source/how-tos/create_an_openedx_app.rst

```diff
@@ -3,6 +3,8 @@ Create an Open edX Application for the Plugin Settings
 
 You will learn how to create an Open edX Application for filling out the form in the Open edX Sync Plugin Settings in our WordPress Settings.
 
+If you are using `Tutor`_ for managing your Open edX instance, you can install `tutor-contrib-wordpress`_ and use its commands to simplify the process.
```

Comment:
```suggestion
If you are using `Tutor`_ for managing your Open edX instance, you can install `tutor-contrib-wordpress`_ and use its commands to assist with the configuration and integration of the Open edX platform with an existing WordPress site.
```

I copied that from the plugin readme. https://github.com/CodeWithEmad/tutor-contrib-wordpress?tab=readme-ov-file#before-you-start